site/gatsby-site/src/components/theme/themeProvider.js

```diff
@@ -28,7 +28,7 @@ class ThemeProvider extends React.Component {
   };
 
   render() {
-    const { children, location } = this.props;
```

Comment:
Why is this being removed? I don't know what it does to begin with.

Reply:
if you checkout Header component, it does not uses the location props at all. so I removed it.